napari/_vispy/layers/base.py

```diff
@@ -131,7 +137,7 @@ def _on_opacity_change(self):
 
     def _on_blending_change(self, event=None):
         blending = self.layer.blending
-        blending_kwargs = BLENDING_MODES[blending].copy()
+        blending_kwargs = BLENDING_MODES[blending].copy()  # type: ignore [attr-defined]
```

Comment:
I prefer (requires cast import):

```suggestion
        blending_kwargs = cast(dict, BLENDING_MODES[blending]).copy()
```